Set/Card/CardViewModel.swift

```diff
@@ -7,9 +7,11 @@ class CardViewModel: ObservableObject {
     }
     
     @Published private var card: CardModel
+    var isColorBlindModeEnabled: Bool
```

Comment:
isColorBlindModeEnabled is constant and `let` should be used

Reply:
fixed